tests/vmcheck/test-cached-rpm-diffs.sh

```diff
@@ -58,6 +58,33 @@ call_dbus() {
     -m org.projectatomic.rpmostree1.OS.$method "$@"
 }
 
+if vm_cmd grep -q 'ID=.*centos' /etc/os-release; then
+  py=python
+else
+  py=python3
+fi
+
+run_transaction() {
+  method=$1; shift
+  sig=$1; shift
+  args=$1; shift
+  cur=$(vm_get_journal_cursor)
+  # use ansible for this so we don't have to think about hungry quote-eating ssh
+  vm_shell_inline <<EOF
+$py -c '
+import dbus
```

Comment:
This is one of the first cases I can think of we're relying on python bindings that live in the host.  Of course Ansible is highly subject to this as well.  And I wouldn't say we should right now start provisioning a tools container...but we should be thinking about it?

Reply:
Yeah, that makes sense overall.
For this test specifically, I'd consider removing it once we do #1305.